pkg/process/checks/process.go

```diff
@@ -22,7 +21,9 @@ import (
 const emptyCtrID = ""
 
 // Process is a singleton ProcessCheck.
-var Process = &ProcessCheck{probe: procutil.NewProcessProbe()}
+var Process = &ProcessCheck{}
```

Comment:
Agreed, in principle, in fact each check already implements the exported `Check` interface and these are currently consumed in [cmd/process-agent](https://github.com/DataDog/datadog-agent/blob/main/cmd/process-agent/collector.go#L74) via the [All](https://github.com/DataDog/datadog-agent/blob/main/pkg/process/checks/checks.go#L20) slice of `Check` checks. If we decide to change the fact that the names are exported, it's probably a change better consolidated to a separate PR because we'd want to rename other checks as well.